redash/query_runner/uptycs.py

```diff
@@ -0,0 +1,136 @@
+from redash.query_runner import *
+from redash.utils import json_dumps
+
+import json
+import jwt
+import datetime
+import requests
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Uptycs(BaseSQLQueryRunner):
+    noop_query = "SELECT 1"
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "url": {
+                    "type": "string"
+                },
+                "customer_id": {
+                    "type": "string"
+                },
+                "key": {
+                    "type": "string"
+                },
+                "SSL": {
+                    "type": "boolean",
+                    "default": True
+                },
+                "secret": {
+                    "type": "string",
+                },
+            },
+            "order": ['url', 'customer_id', 'key', 'secret'],
+            "required": ["url", "customer_id", "key", "secret"],
+            "secret": ["secret", "key"]
+        }
+
+    @classmethod
+    def annotate_query(cls):
+        return False
+
+    def generate_header(self, key, secret):
+        header = {}
+        utcnow = datetime.datetime.utcnow()
+        date = utcnow.strftime("%a, %d %b %Y %H:%M:%S GMT")
+        auth_var = jwt.encode({'iss': key}, secret, algorithm='HS256')
+        authorization = "Bearer %s" % (auth_var)
+        header['date'] = date
+        header['Authorization'] = authorization
+        return header
+
+    def transformed_to_redash_json(self, data):
+        transformed_columns = []
+        rows = []
+        # convert all type to JSON string
+        # In future we correct data type  mapping later
+        if 'columns' in data:
+            for json_each in data['columns']:
+                name = json_each['name']
+                new_json = {"name": name,
+                            "type": "string",
+                            "friendly_name": name}
+                transformed_columns.append(new_json)
+        # Transfored items into rows.
+        if 'items' in data:
+            rows = data['items']
+
+        redash_json_data = {"columns": transformed_columns,
+                            "rows": rows}
+        return redash_json_data
+
+    def api_call(self, sql):
+        # JWT encoded header
+        header = self.generate_header(self.configuration.get('key'),
+                                      self.configuration.get('secret'))
+
+        # URL form using API key file based on GLOBAL
+        url = ("%s/public/api/customers/%s/query" %
+               (self.configuration.get('url'),
+                self.configuration.get('customer_id')))
+
+        # post data base sql
+        post_data_json = {"query": sql}
+
+        response = requests.post(url, headers=header, json=post_data_json,
+                                 verify=self.configuration.get('SSL'))
```

Comment:
```suggestion
                                 verify=self.configuration.get('SSL', True))
```

(default value is True)

Reply:
Fixed.